docs/resources/security_monitoring_suppression.md

```diff
@@ -30,3 +41,12 @@ Provides a Datadog Security Monitoring Suppression API resource. It can be used
 ### Read-Only
 
 - `id` (String) The ID of this resource.
+
+## Import
+
+Import is supported using the following syntax:
+
+```shell
+# Security monitoring suppressions can be imported using ID, e.g.
```

Comment:
```suggestion
# Security monitoring suppressions can be imported using ID, for example:
```

Reply:
Done in [e4e6107](https://github.com/DataDog/terraform-provider-datadog/pull/2288/commits/e4e6107db763b99e2807feb8bd3ce1398ffd9a9c)